client/public/index.html

```diff
@@ -12,6 +12,10 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP&family=Roboto:wght@300&display=swap" rel="stylesheet"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap" rel="stylesheet"> 
```

Comment:
From what I see in Figma, Inter is the main font used with many different weights, not just 500. So I'm not sure why you also have Noto Sans JP and Roboto, and only Inter at 500.

I'd remove Noto Sans JP and Roboto to reduce load time, and make sure you're loading all the variants of Inter needed.